mtp_api/apps/performance/prediction.py

```diff
@@ -68,49 +77,73 @@ def get_value(self, x):
 
 
 known_curves = {
-    'accurate_credits_by_mtp': {
+    'accurate_credits_by_mtp_with_private_estate': {
+        'curve': Logarithmic,
+        'default_params': np.array([83089128.91457231, 1.6418543562686948e-05, 1.000494203778578], dtype='float64')
+    },
+    'accurate_credits_by_mtp_without_private_estate': {
         'curve': Logarithmic,
-        'defaults': np.array([109004.08604805294, 0.022066612129535555, 1.4388566286122846], dtype='float64')
+        'default_params': np.array([78402589.76473396, 1.656958576241639e-05, 1.0005337542137802], dtype='float64')
```

Comment:
Any reason why you're not just setting the default coefficients to unity since you're running the optimisation each time. It feels like you're increasing the risk of finding local minima if you're priming with previous coefficients?

Reply:
as it happens, the optimisation is never run server-side so this is the only place prediction curve values exist